generator/resources/eks_daemon_test_matrix.json

```diff
@@ -1,16 +1,5 @@
 [
   {
-    "k8s_version": "1.24",
-    "test_dir": "./test/metric_value_benchmark"
```

Comment:
There is no need to list test dirs as objects in the matrix, since `test_case_generator` will generate them https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/generator/test_case_generator.go#L95

Reply:
It could have been that those test cases were commented out in the generator itself for faster testing